CMakeLists.txt

```diff
@@ -187,3 +187,57 @@ if(WIN_BUILD)
     set(ENV{PKG_CONFIG_PATH} "${OUTPUTDIR}")
 	message("PKG_CONFIG_PATH: " $ENV{PKG_CONFIG_PATH})
 endif()
+
+# Windows installer build
+if(WIN_BUILD AND RELEASE)
+
+    # IPMCTL release link, used to download IPMCTL Windows installer
+    SET(IPMCTL_RELEASE_URL https://github.com/intel/ipmctl/releases/download)
+    # VERSION NUMBER - REMEMBER TO UPDATE WHEN NEW RELEASE COMES OUT
+    SET(IPMCTL_VERSION 02.00.00.3852)
+    SET(IPMCTL_NAME "ipmctl_windows_install-${IPMCTL_VERSION}.exe")
```

Comment:
Take a look at calls naming convention used across the file, basically we use lower case letter for calls like "set" same for other calls like for instance "message" (line 202), also I noticed "FILE" call instead of "file" (lines 121 and 128) please align all of them to follow one rule across the file.